packages/elements-web-components/src/index.ts

```diff
@@ -1,4 +1,10 @@
+import { dom, library } from '@fortawesome/fontawesome-svg-core';
+import { faChevronDown, faChevronRight, faCube } from '@fortawesome/free-solid-svg-icons';
```

Comment:
Do we list these as `devDependencies` of `elements-web-components`?

Reply:
No, these are direct dependencies in `elements`

Reply:
@marcelltoth Should we?